### ISSUE TITLE
map_server: contains library with generic name "image_loader"

### ISSUE THREAD

**Issue by xqms** [ros-planning/navigation#208]
Hi,

on my system (Ubuntu 14.04) map_server (from debian packages) crashes on startup with:

```
/opt/ros/indigo/lib/map_server/map_server: symbol lookup error: /opt/ros/indigo/lib/map_server/map_server: undefined symbol: _ZN10map_server15loadMapFromFileEPN8nav_msgs15GetMapResponse_ISaIvEEEPKcdbddPdb
```

It's somehow related to the runtime-path stripping in the CMake installation step. I tried to built map_server by hand and had the same issue. I finally had to copy the unstripped executable to `/opt/ros/indigo/lib/map_server` to get it to work.

**Comment by mikeferguson** [comment]
I am unable to reproduce this -- is there something else about your system that is maybe funky?

**Comment by xqms** [comment]
Never mind, I found someone using the name `image_loader` for a library in our project, which clashed with `libimage_loader.so` from map_server. When I used the repository version of map_server, it chose the wrong library at runtime, because the runtime path was stripped.

Maybe we could prefix the library with `map_` to reduce the chance of collisions like these?

Anyway, I'm closing this since it's solved for me.
